kraken-std/src/kraken/std/python/settings.py

```diff
@@ -178,6 +182,13 @@ def python_settings(
     if tests_directory is not None:
         settings.tests_directory = Path(tests_directory)
 
+    if additional_directories is not None:
+        settings.additional_directories = [
+            path
+            for additional_directory in additional_directories
+            if (path := Path(additional_directory)).exists() and path.is_dir()
```

Comment:
It won't give you a neat one-liner, but perhaps we should print paths which do not exist or are not directories?

Reply:
Good shout, done :)